scripts/make-mini-d3.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
+(set -o igncr) 2>/dev/null && set -o igncr; # For Cygwin on Windows compatibility
```

Comment:
igncr is a Cygwin only setting which causes Cygwin bash to recognize \n\r as line endings as opposed to recognising the purely Linux \n as a line ending and then \r as an error.

This line attempts to set the option, suppresses any error messages. If it is not set successively then abort. If there is no error, then the option is set for real.

The snippet in question is from Eric Blake on the Cygwin mailing list from a few years back https://cygwin.com/ml/cygwin-announce/2010-08/msg00015.html this solution particularly seemed like the most painless to implement.